README.md

```diff
@@ -19,9 +19,11 @@ OpenSSL support. Open up a `phpinfo();` page and verify that under the curl
 section, there's a line that says something like:
 
 ```
-libcurl/7.19.5 OpenSSL/0.9.8g zlib/1.2.3.3 libidn/1.15
+libcurl/7.19.5 OpenSSL/1.0.1g zlib/1.2.3.3 libidn/1.15
 ```
 
+Please ensure that your OpenSSL version supports TLS v1.1 or higher. You can view the change log for OpenSSL here: https://www.openssl.org/news/changelog.html.
```

Comment:
That link doesn't work with the `.` on the end. I pulled it up and I've got no idea how I'd determine which versions support it. Can you dig through there and come up with a list?